docs/5.3 Mana.md

```diff
@@ -4,7 +4,7 @@
 
 This section introduces  *access Mana* and *consensus Mana* and determines which modules use each of these two types of Mana.
 
-Any permissionless system needs a Sybil protection mechanism. In Coordicide, this is done by forcing every node to create a node identity (see also [Section 3.3 - Peer Discovery](./3.3%20Peer%20Discovery)). Since the creation of an arbitrarily large number of identities is not an expensive operation, two *Difficult-to-obtain* resources are linked to each node identity; we call them *access Mana*(**aMana**) and *consensus Mana* (**cMana**). Both kinds of Mana can be described as essential resources to multiple parts of the network. They are related to the IOTA token, but are not tokens by themselves <!--(????? add a good explanation here)--> and do not interfere with the token balance in any direct way. When a transaction is processed, a certain amount of aMana and cMana—dependent on the amount of IOTAs moved by the transaction—will be pledged to nodes specified in the transaction (their node IDs must be specified as defined in [Section 2.3 - Standard Payloads Layout](./2.3%20Standard%20Payloads%20Layout)). The access and consensus Mana pledged to each node ID must be stored as an extension of the ledger state. The only way a node can obtain aMana or cMana is to convince some ken holders to pledge to it. Both kinds of Mana provides adequate Sybil protection because they are difficult and costly to be acquired in arbitrarily large amounts.  
+Any permissionless system needs a Sybil protection mechanism. In Coordicide, this is done by forcing every node to create a node identity (see also [Section 3.3 - Peer Discovery](./3.3%20Peer%20Discovery)). Since the creation of an arbitrarily large number of identities is not an expensive operation, two *Difficult-to-obtain* resources are linked to each node identity; we call them *access Mana* (**aMana**) and *consensus Mana* (**cMana**). Both kinds of Mana can be described as essential resources to multiple parts of the network. They are related to the IOTA token, but are not tokens by themselves <!--(????? add a good explanation here)--> and do not interfere with the token balance in any direct way. When a transaction is processed, a certain amount of aMana and cMana—dependent on the amount of IOTAs moved by the transaction—will be pledged to nodes specified in the transaction (their node IDs must be specified as defined in [Section 2.3 - Standard Payloads Layout](./2.3%20Standard%20Payloads%20Layout)). The access and consensus Mana pledged to each node ID must be stored as an extension of the ledger state. The only way a node can obtain aMana or cMana is to convince some ken holders to pledge to it. Both kinds of Mana provides adequate Sybil protection because they are difficult and costly to be acquired in arbitrarily large amounts.  
```

Comment:
```suggestion
Any permissionless system needs a Sybil protection mechanism. In Coordicide, this is done by forcing every node to create a node identity (see also [Section 3.3 - Peer Discovery](./3.3%20Peer%20Discovery)). Since the creation of an arbitrarily large number of identities is not an expensive operation, two *Difficult-to-obtain* resources are linked to each node identity; we call them *access Mana* (**aMana**) and *consensus Mana* (**cMana**). Both kinds of Mana can be described as essential resources to multiple parts of the network. They are related to the IOTA token, but are not tokens by themselves <!--(????? add a good explanation here)--> and do not interfere with the token balance in any direct way. When a transaction is processed, a certain amount of aMana and cMana—dependent on the amount of IOTAs moved by the transaction—will be pledged to nodes specified in the transaction (their node IDs must be specified as defined in [Section 2.3 - Standard Payloads Layout](./2.3%20Standard%20Payloads%20Layout)). The access and consensus Mana pledged to each node ID must be stored as an extension of the ledger state. The only way a node can obtain aMana or cMana is to convince some ken holders to pledge to it. Both kinds of Mana provides adequate Sybil protection because they are difficult and costly to be acquired in arbitrarily large amounts.  
```
```suggestion
Any permissionless system needs a Sybil protection mechanism. In Coordicide, this is done by forcing every node to create a node identity (see also [Section 3.3 - Peer Discovery](./3.3%20Peer%20Discovery)). Since the creation of an arbitrarily large number of identities is not an expensive operation, two *Difficult-to-obtain* resources are linked to each node identity; we call them *access Mana* (**aMana**) and *consensus Mana* (**cMana**). Both kinds of Mana can be described as essential resources to multiple parts of the network. They are related to the IOTA token, but are not tokens by themselves <!--(????? add a good explanation here)--> and do not interfere with the token balance in any direct way. When a transaction is processed, a certain amount of aMana and cMana—dependent on the amount of IOTAs moved by the transaction—will be pledged to nodes specified in the transaction (their node IDs must be specified as defined in [Section 2.3 - Standard Payloads Layout](./2.3%20Standard%20Payloads%20Layout)). The access and consensus Mana pledged to each node ID must be stored as an extension of the ledger state. The only way a node can obtain aMana or cMana is to convince some ken holders to pledge to it. Both kinds of Mana provides adequate Sybil protection because they are difficult and costly to be acquired in arbitrarily large amounts.  
```
```suggestion
Any permissionless system needs a Sybil protection mechanism. In Coordicide, this is done by forcing every node to create a node identity (see also [Section 3.3 - Peer Discovery](./3.3%20Peer%20Discovery)). Since the creation of an arbitrarily large number of identities is not an expensive operation, two *Difficult-to-obtain* resources are linked to each node identity; we call them *access Mana* (**aMana**) and *consensus Mana* (**cMana**). Both kinds of Mana can be described as essential resources to multiple parts of the network. They are related to the IOTA token, but are not tokens by themselves <!--(????? add a good explanation here)--> and do not interfere with the token balance in any direct way. When a transaction is processed, a certain amount of aMana and cMana—dependent on the amount of IOTAs moved by the transaction—will be pledged to nodes specified in the transaction (their node IDs must be specified as defined in [Section 2.3 - Standard Payloads Layout](./2.3%20Standard%20Payloads%20Layout)). The access and consensus Mana pledged to each node ID must be stored as an extension of the ledger state. The only way a node can obtain aMana or cMana is to convince some ken holders to pledge to it. Both kinds of Mana provides adequate Sybil protection because they are difficult and costly to be acquired in arbitrarily large amounts.  
```
```suggestion
Any permissionless system needs a Sybil protection mechanism. In Coordicide, this is done by forcing every node to create a node identity (see also [Section 3.3 - Peer Discovery](./3.3%20Peer%20Discovery)). Since the creation of an arbitrarily large number of identities is not an expensive operation, two *Difficult-to-obtain* resources are linked to each node identity; we call them *access Mana* (**aMana**) and *consensus Mana* (**cMana**). Both kinds of Mana can be described as essential resources to multiple parts of the network. They are related to the IOTA token, but are not tokens by themselves <!--(????? add a good explanation here)--> and do not interfere with the token balance in any direct way. When a transaction is processed, a certain amount of aMana and cMana—dependent on the amount of IOTAs moved by the transaction—will be pledged to nodes specified in the transaction (their node IDs must be specified as defined in [Section 2.3 - Standard Payloads Layout](./2.3%20Standard%20Payloads%20Layout)). The access and consensus Mana pledged to each node ID must be stored as an extension of the ledger state. The only way a node can obtain aMana or cMana is to convince some ken holders to pledge to it. Both kinds of Mana provides adequate Sybil protection because they are difficult and costly to be acquired in arbitrarily large amounts.  
```